dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/enumeration/DssLogLevel.java

```diff
@@ -0,0 +1,22 @@
+package io.github.ztkmkoo.dss.core.network.rest.enumeration;
+import io.netty.handler.logging.LogLevel;
+
+public enum DssLogLevel {
+
+    ERROR(LogLevel.ERROR),
+    WARN(LogLevel.WARN),
+    INFO( LogLevel.INFO),
+    DEBUG( LogLevel.DEBUG),
+    TRACE( LogLevel.TRACE)
+    ;
+
+    private LogLevel level;
+
+    DssLogLevel( LogLevel level) {
+        this.level = level;
+    }
+
+    public LogLevel getNettyLogLevel() {
+        return level;
```

Comment:
These codes are #112 commit. So I think you need to rebase commits! Check please :)

Reply:
Oh I don't know why git consider these codes as new codes after I pulled correctly, I think... Okay, I'll revise this! Thank you for reviewing :>